test/test_mock.py

```diff
@@ -0,0 +1,71 @@
+import yaml
+import github
+import gitlab
+from github.GithubException import UnknownObjectException
+from gitlab.exceptions import GitlabGetError
+
+
+with open('test_githubtest.yaml', 'r') as f:
+    config = yaml.load(f)
+
+
+# github
+
+def mock_get_user(user_name):
+    raise UnknownObjectException('args', 'kwargs')
+
+
+def mock_get_user_(user_name):
+    res = github.NamedUser.NamedUser('args', 'kwargs',
+                                     {'login': user_name}, 'kwargs**')
+    return res
+
+
+def mock_get_repos():
+    repo = github.Repository.Repository('args', 'args*', 'kwargs', 'kwargs*')
```

Comment:
I see how this could create confusion.
So the 'github.Repository.Repository()' method takes 5 arguments. Since I am just mocking the function I am using random arguments to fulfill the requirement of passing 5 arguments.
Would it help if I replace them with 'param1','param2'...'param5'?